test/unit/lifecycle/properties.js

```diff
@@ -41,26 +41,15 @@ describe('lifecycle/property', () => {
       },
     }));
 
-    expect(elem.test1).to.equal('test1');
-    expect(elem.test2).to.equal('test2');
-
-    elem.test1 = null;
-    elem.test2 = null;
-
-    expect(elem.test1).to.equal('test1');
-    expect(elem.test2).to.equal('test2');
-
-    expect(elem.getAttribute('test1')).to.equal(null);
-    expect(elem.getAttribute('test2')).to.equal(null);
-
-    elem.removeAttribute('test1');
-    elem.removeAttribute('test2');
-
-    expect(elem.test1).to.equal('test1');
-    expect(elem.test2).to.equal('test2');
-
-    expect(elem.getAttribute('test1')).to.equal(null);
-    expect(elem.getAttribute('test2')).to.equal(null);
+    ['test1', 'test2'].forEach(value => {
```

Comment:
Already discussed about this, and we shouldn't be able to add a symbol `attribute: true` property. So it doesn't make sense to add a test here with symbol prop. Check #687